modules/compliance-profile-types.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-concepts/compliance-operator-understanding.adoc
+// * security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="compliance_profile_types_{context}"]
+= Compliance Operator profile types
+
+Compliance Operator rules are organized into profiles. Profiles can target the {product-title} Platform or Nodes, and some standards include `rhcos4` Node profiles.
+
+Platform:: Platform profiles evaluate your {product-title} cluster components. For example, confirming APIServer configurations are using strong encryption cyphers is Platform-level rule.
+
+Node:: Node profiles for evaluating the OpenShift or {op-system} configuration of each host. There are `ocp4` Node profiles and `rhcos4` Node profiles. `ocp4` Node profiles evaluate the OpenShift configuration of each host. For example, confirming kubeconfig files have the correct permissions to meet a compliance standard is a Node-level rule. `rhcos4` Node profiles evaluate the {op-system-first} configuration of each host. For example, confirming the SSHD service is configured to disable password logins is a Node-level rule.
+
+[IMPORTANT]
+====
+For benchmarks that have Node and Platform profiles, such as PCI-DSS, you must run both profiles in your {product-title} environment.
+
+For benchmarks that have `ocp4` Platform, `ocp4` Node, and `rhcos4` Node profiles, such as FedRAMP High, you must run all three profiles in your {product-title} environment.
+====
+

```

Comment:
From the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions): "Keep admonitions to a minimum, and avoid placing multiple admonitions close to one another. If multiple admonitions are necessary, restructure the information by moving the less-important statements into the flow of the main content."